packages/email-plugin/src/types.ts

```diff
@@ -133,6 +135,55 @@ export interface SMTPTransportOptions extends SMTPTransport.Options {
     logging?: boolean;
 }
 
+/**
+ * @description
+ * The SES transport options of [Nodemailer](https://nodemailer.com/transports/ses//)
+ *
+ * See [Nodemailers's SES docs](https://nodemailer.com/transports/ses/) for more details
+ * 
+ * @example
+ * ```TypeScript
+ *  import { SES, SendRawEmailCommand } from '@aws-sdk/client-ses'
```

Comment:
due to the way the TS parser works when parsing JSDoc comments, you need to escape the `@` symbol in code blocks otherwise it thinks it is a tag like `@example`. So should read `\@aws-sdk/client-ses` (the escape symbol is removed when generating the docs).